Auth0Tests/AuthenticationSpec.swift

```diff
@@ -284,8 +292,24 @@ class AuthenticationSpec: QuickSpec {
                 }
             }
             
-        }
+            it("should exchange apple auth code for credentials with fullName") {
+                var fullName = PersonNameComponents()
+                fullName.givenName = "John"
```

Comment:
Can you please add test cases for when the name and/or surname are nil?

Reply:
Added.